### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by anki-code**
Screenshot_20201018_035151

### CODE REVIEW

**Comment by anki-code** [comment]
🔴 After merge please change the image path from `docs/_static/xonsh4.png` to absolute path to show image correctly for example in PyPi.

File: README.rst [hunk 1]
```diff
@@ -1,31 +1,28 @@
...
-   :target: https://gitter.im/xonsh/xonsh?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+    <p align="center">
+    <b>xonsh</b> is a Python-powered, cross-platform, Unix-gazing shell language and command prompt.
+    </p>
 
-.. image:: https://img.shields.io/badge/%23xonsh%3Afeneas.org-Matrix-green
-   :alt: Matrix room: #xonsh:feneas.org
-   :target: https://matrix.to/#/#xonsh:feneas.org
+    <p align="center">
+    <a href="https://xon.sh/contents.html#installation"><img src="docs/_static/xonsh4.png" alt="What is xonsh?"></a>
```
Comment (scopatz) [inline README.rst:8]: Maybe use a full URL here, if you want it to work everywhere

**Comment by scopatz** [comment]
Thanks for putting this in @anki-code. A couple of small comments on the image itself. 

* It seems like the word "shell" should be capitalized to "Shell" in the title.
* It is a little strange that the top left box uses ">" as the prompt, while the other three boxes use "$".  It is probably best to use the same symbol for all of the boxes. I would use "$" personally.

**Comment by anki-code** [comment]
Changed:
image

File: README.rst (hunk 1 above)
Comment (anki-code) [reply README.rst]: Done

File: README.rst [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
-   :alt: Matrix room: #xonsh:feneas.org
-   :target: https://matrix.to/#/#xonsh:feneas.org
+    <p align="center">
+    <a href="https://xon.sh/contents.html#installation"><img src="https://raw.githubusercontent.com/xonsh/xonsh/master/docs/_static/xonsh4.png" alt="What is xonsh?"></a>
+    </p>
 
-.. image:: https://travis-ci.org/xonsh/xonsh.svg?branch=master
-    :target: https://travis-ci.org/xonsh/xonsh
+    <p align="center">
+    If you like the idea of xonsh click :star: on the repo and stay tuned by watching releases.
```
Comment (scopatz) [inline README.rst:12]: "If you like xonsh, star the repo and stay tuned by watching releases."

File: README.rst [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
 
-.. image:: https://circleci.com/gh/xonsh/xonsh.svg?style=shield
-    :target: https://circleci.com/gh/xonsh/xonsh
-
-.. image:: https://codecov.io/gh/xonsh/xonsh/branch/master/graph/badge.svg
-    :target: https://codecov.io/gh/xonsh/xonsh
-
-xonsh is a Python-powered, cross-platform, Unix-gazing shell language and command prompt.
-The language is a superset of Python 3.5+ with additional shell primitives.
-xonsh (pronounced *conch*) is meant for the daily use of experts and novices alike.
```
Comment (scopatz) [inline README.rst:26]: This paragraph should still be included somewhere

**Comment by scopatz** [comment]
Thanks @anki-code. I think ` docs/_static/banner-2.jpg` should still be available, for posterity.

File: README.rst [hunk 4]
```diff
@@ -1,31 +1,33 @@
-xonsh
-=====
```
Comment (scopatz) [inline README.rst:2]: It probably makes sens to keep the top-level title

Comment (anki-code) [reply README.rst:2]: First word of the page is `<b>xonsh</b>`. Why we need additional header?

Comment (scopatz) [reply README.rst:2]: I think most documents need titles.  The bold first word is just a nice bonus :wink:

Comment (anki-code) [reply README.rst:2]: May be... for documents... but for people it just looks like visual noise here:
Screenshot_20201019_230632

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**